frontend/src/app/modules/availability/new-slot/notification-emails/notification-emails.component.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable indent */
```

Comment:
Do not use eslint-disable indent

Reply:
Okay, you can return indent: ["error", 2, { "SwitchCase": 1 }]
It would be better to let us know about errors and eslint config changing before operating PR